src/Vue/README.md

```diff
@@ -0,0 +1,14 @@
+# Symfony UX Vue.js
+
+Symfony UX Vue integrates [Vue.js](https://vuejs.org/) into Symfony applications.
+It provides tools to render Vue.js components from Twig.
```

Comment:
Is it worth it to write that it is v3+ ?

Reply:
Will definitely mention it in the documentation, I'll see how I can phrase it to include it here too